src/gui/painting/qpainter.cpp

```diff
@@ -2673,44 +2673,17 @@ QPainterPath QPainter::clipPath() const
     }
 }
 
-/*
-LF-37399
-
-By rounding values to 7dp, we can prevent a special case where
-QPainterPath::intersect returns an empty clipPath when it should not,
-presumably caused by floating point errors.
-
-Seven decimal points was the deepest decimal point to produce the
-correct behaviour, anything above resulted in an incorrect clip path.
-
-An identical Qt bug raised can be found at https://bugreports.qt.io/browse/QTBUG-83102
-*/
-static void round_path_coordinates_to_7_dp(QPainterPath &path) {
-    qreal shifter = pow(10, 7);
-
-    for (int i = 0; i < path.elementCount(); i++) {
-        QPainterPath::Element coordinate = path.elementAt(i);
-        qreal x = qRound64(coordinate.x * shifter) / shifter;
-        qreal y = qRound64(coordinate.y * shifter) / shifter;
-        path.setElementPositionAt(i, x, y);
-    }
-}
 
 static QPainterPath pathClipReplaceClip(const QTransform &matrix, const QPainterClipInfo &info)
 {
-    QPainterPath tempPath = info.path * matrix;
-
-    round_path_coordinates_to_7_dp(tempPath);
-    return tempPath;
+	return info.path * matrix;
```

Comment:
Is there something funny going on with the indentation here? Maybe tabs and spaces issues? Just looks a bit off

Reply:
Yes I pushed using tabs again :/